Lib/test/test_exceptions.py

```diff
@@ -50,7 +50,7 @@ def raise_catch(self, exc, excname):
             self.assertEqual(buf1, buf2)
             self.assertEqual(exc.__name__, excname)
 
-    def testRaising(self):
+    def test_raising(self):
```

Comment:
Let's not rename all the other methods in this PR (as @ericvsmith asked for in the previous version).

Reply:
Ok, I am confused :) I personally don't care about these names one bit, I can rename, or not rename, or rename in a separate PR, or in a separate commit in this PR (I thought that's what was asked). What should I do?

Reply:
Based on https://github.com/python/cpython/pull/30753#discussion_r789849171, you should not rename any unrelated test methods in this PR.

Reply:
Removed that.